### PR TITLE
Project setup - Milestone 1

### PR DESCRIPTION
**Pull request by phfdonda**
This is the First Milestone for the Tic-Tac-Toe Game Project

### CODE REVIEW

File: main [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env ruby
+p 'Hello World'
```
Comment (shshamim63) [inline main:2]: These lines should be placed inside bin/main.rb file.

File: .stickler.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# add the linters you want stickler to use for this project
+linters:
+  stylelint:
+    # indicate where is the config file for stylelint
+    config: 'stylelint.config.js'
```
Comment (shshamim63) [inline .stickler.yml]: The config should be  "./rubocop.yml"

**Review by shshamim63 — CHANGES_REQUESTED** [review]
## Status :trackball: Changes Required :trackball: 
___
Hi @phfdonda :wave: ,
Awesome job :clap: . I can see a bin folder with a main.rb file, a lib folder, the skeleton of a Readme . But have found some issues that are blocking me to approve it right now.
___
### :spades: Mandatory Changes :spades: 
___
1. We want to check the lint based on rubocop specification. For that, you have to change the stickler. Check out this [link](https://github.com/microverseinc/linters-config/blob/master/ruby/.stickler.yml) for more info.
2. `main.rb` should be an [executable file](https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/):
    - `#!/usr/bin/env ruby` is the top line
    -  puts "Hello world" in the following line
Instead saw those lines on main. So you have to move them.
___
##### Please fix the above-mentioned issues and send another review request.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain

**Comment by phfdonda** [comment]
Hi, @shshamim63! Thank you for the review! Silly mistake, copied the wrong file! There you have it!

**Review by pelzolga123 — APPROVED** [review]

**Comment by pelzolga123** [comment]
## Hi Pedro, amazing job :+1:
You did all changes required previously. Just one more thing, you have 2 main files in your project. Please delete one, that is outside bin directory. Then go ahead and merge to development.
 
_Goodluck in your next milestone_ *Happy coding*

**Pull request merged**